utils/doclint/check_public_api/index.js

```diff
@@ -176,8 +183,12 @@ function compareDocumentations(actual, expected) {
     const actualMethods = Array.from(actualClass.methods.keys()).sort();
     const expectedMethods = Array.from(expectedClass.methods.keys()).sort();
     const methodDiff = diff(actualMethods, expectedMethods);
-    for (const methodName of methodDiff.extra)
+    for (const methodName of methodDiff.extra) {
+      const expectedMissing = expectedMissingMethods.find(method => method.className === className && method.methodName === methodName);
```

Comment:
Let's make `expectedMissingMethods` a `Map<String, Set>` to make this more efficient (`O(2)` i.e. `O(1)` instead of `O(n)`).

```js
const expectedMissingMethods = new Map([
  ['Page', new Set('emulateMedia')],
]);
```

Reply:
👍